sdk/include/psdk/versionhelpers.h

```diff
@@ -43,93 +43,111 @@ IsWindowsVersionOrGreater(WORD wMajorVersion, WORD wMinorVersion, WORD wServiceP
 }
 
 VERSIONHELPERAPI
-IsWindowsXPOrGreater()
+IsWindowsXPOrGreater(void)
```

Comment:
```suggestion
IsWindowsXPOrGreater(VOID)
```
Same as for other prototype calls.